sdk/core/azure-core/src/main/java/com/azure/core/util/DateTimeRfc1123.java

```diff
@@ -3,15 +3,21 @@
 
 package com.azure.core.util;
 
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.DateTimeException;
 import java.time.OffsetDateTime;
 import java.time.ZoneId;
+import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
 import java.util.Locale;
 
 /**
  * Wrapper over java.time.OffsetDateTime used for specifying RFC1123 format during serialization and deserialization.
  */
 public final class DateTimeRfc1123 {
+    private final ClientLogger logger = new ClientLogger(DateTimeRfc1123.class);
```

Comment:
Given the performance implications of this class (highly instantiated), I think we should use a static `ClientLogger`